templates/org/aac/aac-design-pattern.md

```diff
@@ -1,95 +1,79 @@
 ---
-title: <Replace this text based on the guidance in Completing the metadata under the Article Title section.>   
-description: <Replace this text based on the guidance in Completing the metadata under the Article Title section.>
-keywords: <Replace this text based on the guidance in Completing the metadata under the Article Title section.>
+title: <Article Title in Title Casing> 
+description: <Align with the template instructions for this content type.>
+keywords: <Align with the template instructions for this content type.>
 ms.date: <Date of publish or last freshness pass, in mm/dd/yyyy format>
 ms.topic: design-pattern
 ms.service: azure-architecture-center
 ms.subservice: design-pattern
 pnp.series.title: Cloud Design Patterns
-pnp.pattern.categories: <Replace this text based on the guidance in Completing the metadata under the Article Title section.>
+pnp.pattern.categories: <Align with the template instructions for this content type.>
```

Comment:
@ckittel - Could you take another look at this metadata and let me know if you want me to make it more like the metadata for aac-architecture-guide-content.md?

Reply:
Yes, and sorry I missed that on review.  This should just be basically the same as 'guide' -- only, with the pattern-specific values for `subservice` and `topic`